src/main/kotlin/no/nav/syfo/dialogmelding/status/domain/DialogmeldingStatus.kt

```diff
@@ -71,3 +72,11 @@ data class DialogmeldingStatus private constructor(
         )
     }
 }
+
+fun DialogmeldingStatus.toKafkaDialogmeldingStatusDTO(): KafkaDialogmeldingStatusDTO = KafkaDialogmeldingStatusDTO(
+    uuid = this.uuid.toString(),
+    createdAt = this.createdAt,
+    status = this.status.name,
+    tekst = this.tekst,
+    bestillingUuid = this.bestilling.uuid.toString(),
```

Comment:
Trenger vi noen flere felter fra bestillingen her eller er det nok med `bestillingUuid` for at konsumentene skal kunne finne hvilken dialogmelding statusen gjelder?

Reply:
Denne uuid'en skal jo referere til en instans av melding i isbehandlerdialog feks, så det burde være tilstrekkelig. Tilsvarende for isdialogmote (dvs refererer til et varsel der)

Reply:
Det lages en ny `DialogmeldingStatus SENDT` når meldingen blir (forsøkt) sendt så da burde det være nok å sende med `createdAt` fra `DialogmeldingStatus` slik det gjøres nå.

Reply:
Ja, akkurat: glemte at dette kommer fra statusen og ikke bestillingen. createdAt på status-raden og sendt-feltet i bestillingen bør jo være praktisk talt det samme tidsstemplet 👍